src/main.ts

```diff
@@ -76,6 +78,6 @@ fetchConfig("config.json").then((config: any) => {
   app.config.globalProperties.$lokapi = lokApiService
   app.config.globalProperties.$config = config
   app.config.globalProperties.$msg = ToastService
-
+  app.config.globalProperties.$persistantStore = new LocalStore("monujo")
```

Comment:
@SeddikKadi Arf you changed the name and there now is a misspelling... :-/

"Persistant" is spelled "persistent" in english.

But as I said I was fine with the other name which was shorter and that trait indeed has some value. My comment was just for my own education. I was not asking you to change it, please read my initial comment carefully.

And since you want to change it, for me the ideal name would make the coder understand that it is localstorage we are talking about. So something like "localData" or event "localStore" since that's what the used lib's constructor is named.

At least please fix the misspelling. Sorry about the confusion.

Reply:
@seegwen i named it just by correcting the spelling. thank you for the review !